public/loading.css

```diff
@@ -0,0 +1,87 @@
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 100%;
+    z-index: 1;
+    display: block;
+    height: 10rem;
+    width: 10rem;
+}
+
+.loading-animation + h2 {
+    margin-top: 4rem;
+    padding-bottom: 8rem;
+    text-transform: uppercase;
+    font-size: 1.75rem;
+    font-weight: 600;
+    line-height: 0.86;
+    letter-spacing: 0.143em;
+}
+
+.logo-container {
+    width: 100%;
+    padding: 4rem 3rem;
+    box-sizing: border-box;
+}
+
+.logo {
+    height: 3.625rem;
+    box-sizing: border-box;
+    font-size: 2.125rem;
+    font-weight: 600;
+    text-transform: uppercase;
+    letter-spacing: 0.077em;
+    display: inline-flex;
+    align-items: center;
+    color: var(--nimiq-blue);
+    z-index: 1;
+    text-decoration: none;
+    user-select: none;
+    -webkit-tap-highlight-color: transparent;
+    outline: none !important;
+}
+
+.logo .nimiq-logo {
+    height: 4rem;
+    width: 4rem;
+    margin-right: 2rem;
+}
+
+.nq-icon {
```

Comment:
When you move the `.nq-*` rules further up, you can maybe reduce some of the class-specificity for the logo.

Reply:
I don't really get it.